server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -568,11 +568,15 @@
   private int primitiveArraySize = 64;
 
   /**
-   * whether enable data partition
-   * if disabled, all data belongs to partition 0
+   * whether enable data partition if disabled, all data belongs to partition 0
```

Comment:
whether to enable data partition. If disabled, all data belongs to partition 0

Reply:
Sorry for the wrong format. Fixed